core/controllers/question.py

```diff
@@ -41,3 +42,82 @@ def get(self):
         return self.render_json({
             'questions_dict': questions_dict
         })
+
+
+class QuestionsHandler(base.BaseHandler):
+    """This handler completes PUT/DELETE requests for questions."""
+    REQUIRE_PAYLOAD_CSRF_CHECK = False
+
+    @acl_decorators.open_access
```

Comment:
Why is this open access? Shouldn't it require login at the very least?

Ditto elsewhere. I think, for now, restrict these handlers to admin/moderator level since the system is still in beta. Safer, and prevents changes to the questions datastore by random users before it's ready for full use. Add a backend test to ensure that users with permissions below this level can't use these handlers.

Reply:
updated.